test/storage/index.js

```diff
@@ -0,0 +1,116 @@
+const { config, run } = require('hardhat');
```

Comment:
Furthermore, can we add this run to the build to run automatically next to unit and integration tests?

As opposed to the migration test this doesn't need a network connection to a fork node so it should run easily.